### ISSUE TITLE
Probably wrong End portal behavior (can't build from the middle but can break some blocks)

### ISSUE THREAD

**Issue by kay27** [kay27/MineClone2#3]
![image](https://user-images.githubusercontent.com/2028802/94587964-5b977500-0294-11eb-9050-82ca12a99a1d.png)

**Comment by anon55555** [comment]
Afaik in Minecraft the portal blocks are pointable and breaking the frames doesn't remove the portal.

**Comment by Wah7** [comment]
> Afaik in Minecraft the portal blocks are pointable and breaking the frames doesn't remove the portal.

yes this is true i play minecraft a lot. this should be kept as a feature so it can be more accurate to minecraft
